src/cuda/cuda-device.cpp

```
@@ -205,6 +205,13 @@ Result DeviceImpl::initialize(const DeviceDesc& desc, BackendImpl* backend)
     addFeature(Feature::RealtimeClock);
     // Not clear how to detect half support on CUDA. For now we'll assume we have it
     addFeature(Feature::Half);
+    addFeature(Feature::Double);
```

Cppcheck (2.20.0)

[style] The function 'makeFromDouble' is never used.

(unusedFunction)